docs/source/gbd2017_models/causes/neoplasms/bladder_cancer/cancer_model.rst

```diff
@@ -0,0 +1,96 @@
+.. _2017_cancer_model_bladder_cancer:
+
+==============
+Bladder Cancer
+==============
+
+Disease Overview
+----------------
+
+.. todo::
+
+   Add definition of each cancer. In particular, find data about global prevalence and disease fatal and non fatal description.
+
+
+GBD 2017 Modeling Strategy
+--------------------------
+
+Bladder cancer in GBD 2017
+++++++++++++++++++++++++++
+
+.. todo::
+
+   Add GBD modelling strategy of bladder cancer.
+
+Cause Hierarchy
+++++++++++++++++
+
+.. todo::
+
+   Add hierarchy diagram.
+
+
+Restrictions
+++++++++++++
+
+.. todo::
+
+   Add restrictions table.
+
+
+Vivarium Modeling Strategy
+--------------------------
+
+
+Scope
++++++
+
+.. todo::
+
+   Add scope.
+
+Model Assumptions and Limitations
++++++++++++++++++++++++++++++++++
+
+.. todo::
+
+   Add assumptions and limitations.
+
+
+Cause Model Diagram
+-------------------
+
+According to GBD 2017, cancer cases are considered cured if the patient survives more than 10 years. But, there is no data available for remission rate 
```

Comment:
I would consider clarifying this language. does this mean that GBD uses a duration-based remission rate? I would also clarify between "according to gbd, the world says.." vs "within GBD, this is defined as..."

Reply:
No, they don't. They have used a complicated model. I will change the wording.